doc/source/user_guide/indexing.rst

```diff
@@ -1870,25 +1870,35 @@ This is the correct access method:
 
 .. ipython:: python
 
-   dfc = pd.DataFrame({'A': ['aaa', 'bbb', 'ccc'], 'B': [1, 2, 3]})
-   dfc.loc[0, 'A'] = 11
-   dfc
+   dfc = pd.DataFrame({'a': ['one', 'one', 'two',
+                             'three', 'two', 'one', 'six'],
+                       'c': np.arange(7)})
+   dfd = dfc.copy()
+         
```

Comment:
Some extra whitespace around here that should be removed

Reply:
fixed in https://github.com/pandas-dev/pandas/pull/34461/commits/1bd5d9daa4add2f10bbb959d7e5a6b42df191532